tests/conftest.py

```diff
@@ -31,9 +32,10 @@
 
 @pytest.fixture(autouse=True)
 def test_reset():
-    """Reset global state before all tests."""
+    """Reset global state before every test."""
     flags.verbosity = 0
     flags.cylc7_back_compat = False
+    LOG.setLevel(logging.NOTSET)
```

Comment:
Yes, I've been having xdist fun with the skip mode PR.

Reply:
Renamed to `before_each()`. I don't know if it makes sense to do this as a teardown instead of setup. I think usually you would do resetting of state in setup, and cleanup of resources in teardown.